roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -23,33 +28,156 @@ The meta information that we track per-candidate is defined as the `CandidateVot
 This draws on the [dispute statement types][DisputeTypes]
 
 ```rust
-struct CandidateVotes {
-    // The receipt of the candidate itself.
-    candidate_receipt: CandidateReceipt,
-    // Sorted by validator index.
-    valid: Vec<(ValidDisputeStatementKind, ValidatorIndex, ValidatorSignature)>,
-    // Sorted by validator index.
-    invalid: Vec<(InvalidDisputeStatementKind, ValidatorIndex, ValidatorSignature)>,
+/// Tracked votes on candidates, for the purposes of dispute resolution.
+#[derive(Debug, Clone, Encode, Decode)]
+pub struct CandidateVotes {
+	/// The receipt of the candidate itself.
+	pub candidate_receipt: CandidateReceipt,
+	/// Votes of validity, sorted by validator index.
+	pub valid: Vec<(ValidDisputeStatementKind, ValidatorIndex, ValidatorSignature)>,
+	/// Votes of invalidity, sorted by validator index.
+	pub invalid: Vec<(InvalidDisputeStatementKind, ValidatorIndex, ValidatorSignature)>,
 }
 
-// The status of the dispute.
-enum DisputeStatus {
-  // Dispute is still active.
-  Active,
-  // Dispute concluded positive (2/3 supermajority) along with what
-  // timestamp it concluded at.
-  ConcludedPositive(Timestamp),
-  // Dispute concluded negative (2/3 supermajority, takes precedence over
-  // positive in the case of many double-votes).
-  ConcludedNegative(Timestamp),
+/// The mapping for recent disputes; any which have not yet been pruned for being ancient.
+pub type RecentDisputes = std::collections::BTreeMap<(SessionIndex, CandidateHash), DisputeStatus>;
+
+/// The status of dispute. This is a state machine which can be altered by the
+/// helper methods.
+#[derive(Debug, Clone, Copy, Encode, Decode, PartialEq)]
+pub enum DisputeStatus {
+	/// The dispute is active and unconcluded.
+	#[codec(index = 0)]
+	Active,
+	/// The dispute has been concluded in favor of the candidate
```

Comment:
```suggestion
	/// The dispute was concluded in favor of the candidate
```

Reply:
This comment is copy-pasted from the code. I'd better remove it altogether because it doesn't contribute to understanding how the subsystem works.